content/docs/authorization/cli.md

```diff
@@ -21,7 +21,7 @@ If you feel that something is unclear or missing in this document, please open u
 | [karavictl role get](#karavictl-role-get) | Get role |
 | [karavictl role list](#karavictl-role-list) | List roles |
 | [karavictl role create](#karavictl-role-create) | Create one or more CSM roles |
-| [karavictl role update](#karavictl-role-update) | Update one or more CSM roles |
+| [karavictl role update](#karavictl-role-update) | Update the quota one or more CSM roles |
```

Comment:
Should this say "update the quota **of** one ..."?

Reply:
Yes, fixed.